VERSION

```diff
@@ -1 +1 @@
-0.2.0
\ No newline at end of file
+0.3.0
```

Comment:
Don't modify this file since

1. it is generated by scripts.
2. it hasn't been released.